package.json

```diff
@@ -88,5 +90,10 @@
   },
   "resolutions": {
     "react": "17.0.2"
-  }
+  },
+  "sideEffects": [
```

Comment:
without this, ui-lib is not tree-shakeable. even if components are separated, they are all included in AP main bundle.
if this is set to false, ui-lib is tree-shakeable, but it won't have any css